libc/test/src/__support/integer_literals_test.cpp

```diff
@@ -0,0 +1,134 @@
+
+//===-- Unittests for user defined integer literals -----------------------===//
```

Comment:
Do we need any cmake additions to be able to build+run this test via cmake?

Reply:
ah, consider using the "convert to draft" button in github UI to denote a WIP PR.